python/lsst/cp/pipe/defects.py

```diff
@@ -142,6 +143,19 @@ class FindDefectsTaskConfig(pexConfig.Config):
              "if the run key isn't found."),
         default=False,  # false because most obs_packages don't have runs. obs_lsst/ts8 overrides this.
     )
+    ignoreFilters = pexConfig.Field(
+        dtype=bool,
+        doc=("Set the filters used in the CALIB_ID to NONE regardless of the filters on the input"
+             " images. Allows mixing of filters in the input flats. Set to True if you think"
```

Comment:
If the default is True, then this docstring should note that the user should "Set to False if you think your defects are not chromatic" etc.  My actual suggestion is to make the default False, because I would prefer not to think about chromatic defects.

Reply:
I agree with the first part, and for the default to be not supporting chromatic defects, but I therefore think you've for a sign flip. It now reads:
`    ignoreFilters = pexConfig.Field(
        dtype=bool,
        doc=("Set the filters used in the CALIB_ID to NONE regardless of the filters on the input"
             " images. Allows mixing of filters in the input flats. Set to False if you think"
             " your defects might be chromatic and want to have registry support for varying"
             " defects with respect to filter."),
        default=True,`

Reply:
I think that is correct now. Shout if I'm mistaken.